csharp/src/Chunk.cs

```diff
@@ -0,0 +1,93 @@
+﻿// This file is part of the ArmoniK project
+//
+// Copyright (C) ANEO, 2022-2022.All rights reserved.
+//   W.Kirschenmann   <wkirschenmann@aneo.fr>
+//   J.Gurhem         <jgurhem@aneo.fr>
+//   D.Dubuc          <ddubuc@aneo.fr>
+//   L.Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F.Lemaitre       <flemaitre@aneo.fr>
+//   S.Djebbar        <sdjebbar@aneo.fr>
+//   J.Fonseca        <jfonseca@aneo.fr>
+//   D.Brasseur       <dbrasseur@aneo.fr>
```

Comment:
Remove all the above lines (names of people) in the new project.
Should be done for all the projects but let'st start here.

Reply:
Done. `.editorconfig` has veen modified directly on main.